### ISSUE TITLE
A11y_Azure Data Studio_Azure Added Account_ColorContrast: Color contrast ratio of Keyboard focus outline is less than 3:1.

### ISSUE THREAD

**Issue by harshgangrade111** [microsoft/azuredatastudio#14169]
“[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags:** #A11yWCAG2.1;#A11yTCS;#A11ySev2;#SH_AzureDataStudio_Feb2021;#SQL Azure Data Studio;#Win10;#FTP;#DesktopApp;#WCAG1.4.11;#keyboard;#ColorContrast;#AINotInScope;

**Environment Details:**
Application Name: Azure Data Studio
Version: 1.26.0-insider (user setup)
Windows Version: Windows10
Tool: Colour Contrast Analyser (CCA) Version 3.1.2

**Repro Steps:**
1.Launch Azure Data Studio insiders application.
2.Tab to Connections from left menu and in servers button and hit enter on "New Connection" button.
3.Connections section will get open.
4.Tab to "Browse Preview" Tab item and hit enter.
5.Navigate to "Azure" tree item with the help of Tab key and press Shift+F10.
6.Select "Azure Signin" Menu Item and hit enter on it.
7.Tab to "Add an account" button and add account.
8."AzureAdded Account" screen is visible now.
9.Tab to "Azure" button present in Azure Added Account section.
10.Start Color Contrast analyser(CCA) tool and analyse color contrast ratio of keyboard focus outline.

**Actual:**
Color contrast ratio of Keyboard focus outline is less 3:1.

**Same issue is also observed with throughout the application.**

**Expected:**
Color contrast ratio of Keyboard focus outline should be more than or equal to 3:1 sop then it will be properly visible.

**User Impact:**
Low vision users will not be able to see keyboard focus properly if color contrast if low..

**Comment by kapilvaishna** [comment]
**GithubTags**:#Closed;
**GithubTags**:#Regressed:08-02-23;
![14169_Closed](https://user-images.githubusercontent.com/124254981/217539988-44234fe1-214c-4100-84fe-8a4cc14f78fa.png)
